packages/models-library/src/models_library/projects_state.py

```diff
@@ -99,6 +91,23 @@ def check_status_compatible(cls, v, values):
             raise ValueError(msg)
         return v
 
+    @validator("owner", always=True)
+    @classmethod
+    def check_owner_compatible(cls, v, values):
+        if (
+            values["value"] is True
+            and v is None
+            and values.get("status")
+            in [
+                status.value
+                for status in ProjectStatus
+                if status != ProjectStatus.MAINTAINING
+            ]
+        ):
+            msg = "Owner must be specified when the project is not in the 'MAINTAINING' status."
```

Comment:
is this compatible with what the GC does. I might remember this guy also does not have an owner no?

Reply:
Yes, GC also puts owner of the lock (probably the last user that was running it)